src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -71,10 +71,18 @@ public override void Shutdown()
 
         public override void Dispose()
         {
-            this.linterMenuItem.Checked -= MenuItemCheckHandler;
-            this.linterMenuItem.Unchecked -= MenuItemUnCheckedHandler;
-            if (linterManager != null) linterManager.PropertyChanged -= OnLinterManagerPropertyChange;
-            viewLoadedParamsReference.ViewExtensionOpenRequest -= OnViewExtensionOpenRequest;
+            if (linterMenuItem != null)
```

Comment:
all these null checks in the extensions are required because during splash screen close - Dynamo is in a strange state - the DynamoView is constructed, but not shown, and so extensions may only have had some of their hooks called. 

We need to make sure that code is more defensive now that splash screen introduces these strange states.

Reply:
make sense

Reply:
Can we use the question mark syntax for these?

Reply:
no, unfortunately not because of the event subscriptions.